pkg/sql/feature_derivation_test.go

```diff
@@ -159,8 +160,7 @@ func TestFeatureDerivation(t *testing.T) {
 	nc, ok = fc3.(*codegen.NumericColumn)
 	a.True(ok)
 
-	// trainIR.Features["feature_columns"][3] is the column class
```

Comment:
Was this a bug? Why was the label column in `trainIR.Features`?

Reply:
The previous implementation includes the label in `trainIR.Features`. Removed now.